File: annotation2-ui/restore_css.py

import os

# Use relative paths assuming cwd is annotation2-ui
dist_path = os.path.join("dist", "assets", "index-BabP2RLM.css")
src_path = os.path.join("src", "App.css")

if not os.path.exists(dist_path):
    print(f"Error: {dist_path} not found")
    exit(1)

with open(dist_path, "r", encoding="utf-8") as f:
    content = f.read()

# Simple formatting to make it readable (optional, but good for future edits)
# Replace '}' with '}\n'
# Replace '{' with ' {\n'
# Replace ';' with ';\n'
# Note: This is naive but works for standard CSS structure.
formatted = content.replace("}", "}\n").replace("{", " {\n").replace(";", ";\n")

with open(src_path, "w", encoding="utf-8") as f:
    f.write(formatted)

print(f"Restored {src_path} from {dist_path}")

File: frontend/restore_css.js

const fs = require('fs');
const path = require('path');

const distPath = path.join('dist', 'assets', 'index-BabP2RLM.css');
const srcPath = path.join('src', 'App.css');

try {
    const content = fs.readFileSync(distPath, 'utf8');
    const formatted = content.replace(/\}/g, '}\n').replace(/\{/g, ' {\n').replace(/;/g, ';\n');
    fs.writeFileSync(srcPath, formatted);
    console.log(`Restored ${srcPath}`);
} catch (err) {
    console.error(err);
    process.exit(1);
}
